ajax/libs/encoding-japanese/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "encoding-japanese",
```

Comment:
Is there special reason to use npm package name as name of this library but use git auto-update?

Reply:
@extend1994 
I just followed the name in files `package.json` and `bower.json` of their githhub repo.
Also I thought it would be better  if we match the name with bower and npm. 
Should it be revised to be as same as the name of the github repo?
Thanks.

Reply:
@cy91244 Since package.json in its git repo is for npm, it makes sense that the name is same as npm package name. For bower.json, it's a similar case.
Not sure if you notice the issue was opened by the author. I would suggest you to ask him which he prefer.

Reply:
@cy91244 @extend1994 I'm encoding.js author.
This package is known under the name `encoding.js`.
I wanted to registered npm with the name `encoding`, but it was already in use, I registered name of `encoding-japanese`.
In this case, I thought that it was better to use the name of `encoding-japanese` according to npm and bower.
Thanks!

Reply:
@extend1994 Yeah, I  should probably check with the author initially. Thanks for the suggestion.
@polygonplanet Thank you for the response! 
So I think we shall just leave the name to be `encoding-japanese` here.